packages/web-app-epub-viewer/src/App.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <main>
+    <app-top-bar :resource="resource" @close="closeApp" />
+    <loading-screen v-if="loading" />
+    <error-screen v-else-if="loadingError" />
+    <div v-else class="oc-height-1-1">
+      <object class="epub-viewer oc-width-1-1" :data="url" type="application/epub" />
```

Comment:
I doubt this works, but I guess you're not finished yet...

Reply:
Can you help me here, I have solved all other problems. I have started localhost:9200 with docker and local cloud working.

Reply:
Well, you need to figure out, how to render an epub file in a browser. I assume you need some js library to render an epub file

Reply:
I have added the js file please check it and installed EPUB with pnpm.
![epubjs](https://user-images.githubusercontent.com/43544330/207548809-a89a98af-90c2-44eb-bca6-98e0625cac12.png)

Reply:
I have no idea what that screenshot is supposed to show me

Reply:
Just adding a dependency to the library is not enough, you need to `import` it in your `App.vue` and make use of it.

You likely need to call `renderTo` in your `mounted()` hook and on file change.